app/assets/stylesheets/fonts.scss

```diff
@@ -1,4 +0,0 @@
-@font-face {
-   font-family: 'Optima';
-   src: url('fonts/unicode.optima.ttf')
-}
```

Comment:
This url was failing to load, causing your assets to crash. I'm pretty sure this is why you weren't getting CSS updates on page refresh (I wasn't either until I deleted this). You don't use the `@font-face` mixin anywhere, so this is safe to delete